apps/workers/sql-load/worker.go

```diff
@@ -76,13 +84,20 @@ func (o *options) newWorker(info string) task.Worker {
 	w := &worker{
 		options: *o,
 		Meta:    task.NewMeta(),
-		ds:      NewDataSet(),
 	}
 
 	if err := uri.Unmarshal(info, &w.Params); err != nil {
 		return task.InvalidWorker("params uri.unmarshal: %v", err)
 	}
 
+	// assume if csv is in the file name, the loading type is csv
+	if strings.Contains(w.Params.FilePath, "csv") {
```

Comment:
do we need to be concerned about files have aren't csv like `csv/*/data.json` we should probably only look at the file extension. Also if we have the csv option in the url maybe we should remove it from the url as an option or only support setting the format via the URL

Reply:
hmmm, that's an interesting question. Maybe we do need a json option.  That kind of path would be confusing, but I could see it as an edge case.  Maybe change the "csv" option to a "file_type" option?